flake-modules/dev/list-plugins.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python3
 
-from enum import Enum
-from dataclasses import dataclass
-from typing import Optional
+import argparse
 import glob
 import re
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
```

Comment:
Is re-sorting the imports an automated change? Is there any way we can add it to our treefmt configuration?

Reply:
Yeah, was automated by whatever python formatter I have installed. I never work with python but looks like I have this defined in conform-nvim
```nix
        python = [
          "isort"
          "black"
        ];
```

Reply:
Ok, looks like we can just add [`treefmt.programs.isort.enable`](https://flake.parts/options/treefmt-nix#opt-perSystem.treefmt.programs.isort.enable) then.

Note: our treefmt is configured with `ruff`, not `black`, so you may run into formatting conflicts and may also want to check you've not re-formatted unrelated code.

Reply:
Sounds good, I'll do that. I'll also switch my formatter to ruff since I don't do python that much either.